### PR TITLE
[SYN5-290] Implement onboarding screen in the new dashboard

### PR DESCRIPTION
**Pull request by adamwardecki**
Styling and basing functionality (modal appears if there are no instances and disappears if there's at least one) is in place. I will still need to work on the modal behavior (currently buggy) and tests.

 
beta_onboarding

### CODE REVIEW

**Comment by gh3r** [comment]
@adamwardecki Also please apply current changes from devel

**Comment by adamwardecki** [comment]
welcome

File: test/e2e/tests/signup_logins/testSignup.js [hunk 1]
```diff
@@ -16,6 +16,6 @@ export default addTestNamePrefixes({
       .fillInput('@passInput', tempPass)
       .clickElement('@submitButton')
       .waitForElementNotPresent('@setupScreen')
-      .assert.urlContains('https://dashboard-old.syncano.rocks/#/');
+      .assert.urlContains('https://localhost:8080/#/instances/');
```
Comment (adamwardecki) [inline test/e2e/tests/signup_logins/testSignup.js]: @Flixow Is this correct. Tests would fail with the 'https://dashboard-old.syncano.rocks/#/' url

**Comment by adamwardecki** [comment]
@gh3r I've added all the changes, please review. The zero state instead of a modal is much better solution, thanks!

I've also removed the setup screen as it wasn't necessary anymore (we don't create an instance so there's no reason to show it).

File: src/apps/Instances/Instances.jsx [hunk 2]
```diff
@@ -102,24 +104,38 @@ const Instances = React.createClass({
...
-          <InstancesList
-            ref="myInstancesList"
-            name="Instances"
-            items={myInstances}
-            isLoading={isLoading}
-            hideDialogs={hideDialogs}
-            emptyItemHandleClick={this.showInstanceDialog}
-            emptyItemContent="Create an instance"
-          />
+          <Show if={myInstances.length}>
```
Comment (Flixow) [inline src/apps/Instances/Instances.jsx]: Can't I see instances list if I have only shared instances?

Comment (adamwardecki) [inline src/apps/Instances/Instances.jsx]: @Flixow There's a separate itemList for the shared instances (line 140)

Comment (Flixow) [inline src/apps/Instances/Instances.jsx]: Oh, okey. I didn't notice 👍

**Pull request merged**